src/mlpack/methods/ann/layer/max_pooling.hpp

```diff
@@ -194,9 +231,22 @@ class MaxPooling
 
         if (!deterministic)
         {
-          arma::Mat<size_t> subIndices = indices(arma::span(rowidx,
+          arma::Mat<size_t> subIndices;
+
+          if (isPadded)
+          {
+            subIndices = paddedIndices(arma::span(rowidx,
+            rowidx + kernelWidth - 1 - offset),
+            arma::span(colidx,
```

Comment:
This is a style error. Remember this is `function call` not `function declaration` so you should wrap it normally (i.e to wrap only if the line exceed 80 width size and that too by two tabs on next line). https://github.com/mlpack/mlpack/wiki/DesignGuidelines

Reply:
Fixed.